launcher/image/containerd.go

```diff
@@ -37,7 +37,45 @@ func (m *containerdManager) Present(imageName string) (bool, error) {
 	return true, nil
 }
 
+func (m *containerdManager) pull(imageName string) error {
+	pullCmd := exec.Command(
+		"nerdctl",
+		"--address",
+		"/clabernetes/.node/containerd.sock",
+		"--namespace",
+		"k8s.io",
+		"image",
+		"pull",
+		"--all-platforms",
+		imageName,
+	)
```

Comment:
[here](https://github.com/containerd/containerd/issues/5895#issuecomment-955005389) I see that there is a command `content fetch` for ctr, that might <?> be more optimal? Since it will just do less work compared to pull (my assumption)